app/services/medium_providers/provider.rb

```
@@ -49,6 +66,26 @@ def validate
 
     private
 
+    def parse_media(media)
+      media = JSON.parse(media)
+      if media.is_a?(Array)
+        media.reject { |medium| is_invalid_hash(medium) }
+      else
+        logger.error("Expected #{entity.name} additional_media parameter to be an array.")
+        []
+      end
+    rescue JSON::ParserError
+      logger.error("JSON parsing error on #{entity.name}'s additional_media parameter.")
```

Oh can you also report the exception through foreman logging `exception` method so it is logged properly with all fields?